.github/workflows/offline-installation.yml

```diff
@@ -0,0 +1,76 @@
+name: Offline installation test
+on:
+  pull_request:
+    paths:
+      - 'unattended_installer/install_functions/wazuh-offline-download.sh'
+      
+jobs:
+  Build-wazuh-install-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          cancel_others: 'true'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          skip_after_successful_duplicate: 'false'
+
+      - uses: actions/checkout@v2
+      - name: Build wazuh-install script and use pre-release packages
+        working-directory: ./unattended_installer
+        run: |
+          bash builder.sh -i -d
+          sed -i 's|wazuh_major="4\.5"|wazuh_major="4\.4"|g' wazuh-install.sh
+          sed -i 's|wazuh_version="4\.5\(.*\)"|wazuh_version="4\.4\1"|g' wazuh-install.sh
+        shell: bash
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: script
+          path: |
+            unattended_installer/wazuh-install.sh
+          if-no-files-found: error
+          
+  Test-offline-installation-debian:
+    runs-on: ubuntu-latest
+    needs: Build-wazuh-install-script
+    steps:
+      - name: Cancel previous runs
```

Comment:
I don't think it is necessary to call the step on every job, just once for each workflow

Reply:
True. Done in https://github.com/wazuh/wazuh-packages/pull/2071/commits/1eda46be9517179d44c28ae854e32b078d880347.